### DISCUSSION TITLE
time and date in grid

### DISCUSSION THREAD

**Discussion by aloispichler** [Alexays/Waybar#2715] [Q&A] [answered]
How/ Is it possible to have time and date in a grid, as in this picture?
![cp](https://github.com/Alexays/Waybar/assets/45781387/b780707c-c257-45a6-85de-6e71d9bc6146)

**Comment by LukashonakV** [comment, marked as answer]
Hi @aloispichler please try (make sure you're using the freshest Waybar):
```json
        "format" : "{:%I:%M %p\n%D}",
```

![ps_2024-01-10-23_56_46](https://github.com/Alexays/Waybar/assets/23121044/a3c41991-092e-49fe-b754-4703fd3e00de)

You can find format masks here : [chrono format](https://omegaup.com/docs/cpp/en/cpp/chrono/format.html)

**Reply by aloispichler** [reply]
Thank you! This helped :-)

**Reply by LukashonakV** [reply]
@aloispichler, please mark QA as done then. Thanks)
